bin/hveto

```diff
@@ -39,11 +37,7 @@ from getpass import getuser
 from distutils.spawn import find_executable
 from pathlib import Path
 
-try:
-    import configparser
-except ImportError:  # python 2.x
-    import ConfigParser as configparser
-
+import configparser
```

Comment:
Can you move this up to sit with the rest of the standard library imports?

Reply:
@duncanmmacleod, this is done in the latest commit.